HIPs/anoncreds.md

```diff
@@ -0,0 +1,171 @@
+* _Will Abramson, Edinburgh Napier University, [will.abramson@napier.ac.uk](mailto:will.abramson@napier.ac.uk)_
+* _Alexander Shcherbakov, DSR Corporation, [alexander.sherbakov@dsr-corporation.com](mailto:alexander.sherbakov@dsr-corporation.com)_
+* _Philippe Bourque, MCN - Québec, [philippe.bourque@mcn.gouv.qc.ca](mailto:philippe.bourque@mcn.gouv.qc.ca)_
+* _Joelson Piaia, MCN - Québec, [Joelson.Piaia@mcn.gouv.qc.ca](mailto:Joelson.Piaia@mcn.gouv.qc.ca)_
+* _Ourida Amrane MCN - Québec [Ourida.Amrane-ext@mcn.gouv.qc.ca](mailto:Ourida.Amrane-ext@mcn.gouv.qc.ca)_
+* _Daniel Bachenheimer, Digital Identity Innovations Technology Lead, Accenture, [daniel.bachenheimer@accenture.com](mailto:daniel.bachenheimer@accenture.com)_
+
+## Abstract
+
+Hyperledger AnonCreds is a type of verifiable credential that supports important privacy-protecting capabilities based on zero-knowledge proof cryptography. The privacy-by-design capabilities of AnonCreds make them particularly important for identity use cases such as those implemented by the combination of [Hyperledger Aries](https://www.hyperledger.org/use/aries) and [Hyperledger Indy](https://www.hyperledger.org/use/hyperledger-indy). The AnonCreds project consumes the cryptographic support provided by [Hyperledger Ursa](https://www.hyperledger.org/use/ursa) to provide secure secret management, decentralized key management and signature scheme functionality.
+
+## Context
+
+Hyperledger AnonCreds is a specification of the AnonCreds verifiable credential data model/interactions, implementations of open source versions of the specification, test suites, and interfaces for publishing and retrieving AnonCreds objects. AnonCreds objects can be published to DLT ledgers, such as Hyperledger Indy or [Fabric](https://www.hyperledger.org/use/fabric), blockchains such as Ethereum (using [Hyperledger Besu](https://www.hyperledger.org/use/besu)) or other Ethereum client implementations, or even centralized systems, such as web servers. AnonCreds verifiable credentials and presentations are passed between issuers, holders and verifiers in an AnonCreds-specific format today, and will use the [W3C Verifiable Credential Data Model Standard v1.1](https://www.w3.org/TR/vc-data-model/) (and greater) in the near future.
+
+Several AnonCreds open source implementations are part of the Hyperledger Indy project, and [one of those implementations](https://github.com/hyperledger/indy-shared-rs/tree/main/indy-credx) will be the basis of a new, ledger-agnostic AnonCreds open source implementation. The implementation will be made agnostic to where the AnonCreds objects are published, still supporting Hyperledger Indy, but also other Verifiable Data Registries (VDRs).
+
+The [AnonCreds specification](https://anoncreds-wg.github.io/anoncreds-spec/) uses the Linux Foundation JDF [Community Specification License v1.0](https://github.com/CommunitySpecification/1.0) and can be safely moved into the Hyperledger project under the same License.
```

Comment:
I'll scan this to make sure this is clear in the doc, but for now:

1. “Community Specification License” is from the Linux Foundation (so unlikely to be a problem) and can be thought of the specification equivalent to an Open Source license.  The idea is that instead of every spec-creation organization defining their IP requirements when collaborating on a spec, we use a common license developed by an organization (LF) that knows what it is doing.  A contributing organization can do a legal review of the License once, and then when joining future spec. organizations that use the same license, they would know what they were getting into.

2. Hyperledger would be for "incubating" the specification. Once completed, the specification might be then submitted to a standards setting organization.  The use of the “Community Specification License” ensures that there will not be IP issues when submitting the specification.

Reply:
Thank you